### PR TITLE
docs(release-highlights): Updates beta highlights.

### LINKED ISSUE #4221: Update PF6 beta release highlights
Currently, there are no call outs to the types of changes made to EmptyState components, which is a pretty impactful change.

Product testers of the PF6 betas were also confused about the apparent removal of `ToolbarItem`'s `variant` prop. Some notes about that change may be helpful as well.

[per this slack thread](https://redhat-internal.slack.com/archives/C04JTH3LQF6/p1724272824821169)

### PR DESCRIPTION
**Pull request by edonehoo**

### CODE REVIEW

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 1]
```diff
@@ -3,37 +3,41 @@ id: Release highlights
...
-- All [tokens](/tokens/about-tokens) pages
+- All tokens pages: 
+    - [About tokens](/tokens/about-tokens)
+    - [All PatternFly tokens](/tokens/all-patternfly-tokens)
+    - [Develop with tokens](/tokens/develop-with-tokens)
+    - [Design with tokens](/tokens/design-with-tokens)
 - Our design foundations 
     - [Colors](/design-foundations/colors) 
     - [Icons](/design-foundations/icons) 
+
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:27]:
```suggestion

```

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 2]
```diff
@@ -3,37 +3,40 @@ id: Release highlights
...
+        
+          
+
+
+        
+        Expand All
+
+    @@ -51,11 +55,19 @@ We also made updates to some of our component structure and naming conventions.
+
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:36]:
```suggestion
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 3]
```diff
@@ -3,37 +3,36 @@ id: Release highlights
...
+    - [Design with tokens](/tokens/design-with-tokens)
 - Our design foundations 
     - [Colors](/design-foundations/colors) 
     - [Icons](/design-foundations/icons) 
+
+        
+          
+
+
+
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:32]:
```suggestion
```

File: packages/ast-helpers/CHANGELOG.md [hunk 4]
```diff
@@ -944,6 +944,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 
 
+<<<<<<< HEAD
```
Comment (edonehoo) [inline packages/ast-helpers/CHANGELOG.md:947]:
```suggestion
```

File: packages/ast-helpers/CHANGELOG.md [hunk 5]
```diff
@@ -1327,6 +1328,8 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 **Note:** Version bump only for package @patternfly/ast-helpers
 
 
+=======
+>>>>>>> 0860b9376 (chore(release): releasing packages [ci skip])
```
Comment (edonehoo) [inline packages/ast-helpers/CHANGELOG.md:1332]:
```suggestion
```

File: packages/documentation-framework/CHANGELOG.md [hunk 6]
```diff
@@ -944,6 +944,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 
 
+<<<<<<< HEAD
```
Comment (edonehoo) [inline packages/documentation-framework/CHANGELOG.md:947]:
```suggestion
```

File: packages/documentation-framework/CHANGELOG.md [hunk 7]
```diff
@@ -1332,6 +1333,8 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 
 
+=======
+>>>>>>> 0860b9376 (chore(release): releasing packages [ci skip])
```
Comment (edonehoo) [inline packages/documentation-framework/CHANGELOG.md:1337]:
```suggestion
```

**Review by edonehoo — COMMENTED** [review]
remove these

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 8]
```diff
@@ -3,37 +3,30 @@ id: Release highlights
 title: Release highlights
 section: get-started
 ---
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:5]:
```suggestion
---

```

**Comment by edonehoo** [comment]
@nicolethoen @dlabaj The issue that this closes also has a comment mentioning "add notes about react-token and variable name changes as well". Do you know if this is referring to something more specific than the tokens table and migration docs we have on staging?

**Comment by nicolethoen** [comment]
@edonehoo linking to the tokens docs is really helpful for sure.

Additionally, maybe under those new links to the new docs, could add even just a couple sentences to call out the fact that any @patternfly/react-tokens referencing global variables will need to be updated since the global variables have been replaced with the tokens. The guidance for how to determine the correct react-token to use to replace the global variable is the same as they were for css / design tokens.

**Pull request closed without merging**